packages/ag-charts-community/src/chart/dom/icons.css

```diff
@@ -0,0 +1,221 @@
+/**
+To create an icon uri:
+    1. Copy .svg file contents and paste markup into https://www.svgviewer.dev/svg-to-data-uri
+    2. Keep but ignore any `fill` attributes (these are overriden by the styles below)
+    3. Press the `Optimize` button
+    4. Copy base64 data uri
+*/
```

Comment:
Worth adding your generator script for these in the comment block? Or do we have a place for these kinds of scripts?